automation/_input/06_Bestiary_SAMPLE.yaml

```diff
@@ -29,12 +29,42 @@ Template:               # must be unique
   Phases:               # Only for bosses
     One:                #
       HP: X             #
-      Allies: X         # List of who appears. If multiple of same, list mult times
+      Allies: X         # List of who appears. If multiple of same, list multiple times
   Description: Descrip  # Full descriptive text
 
+Clubs1:
+  Type: PC
+  Pronouns: "He/Him"
+  Role: Defender
+  Level: 1
+  Primary_Skill: STR
+  HP: 7
+  AP: 2
+  PP: 2
+  Attribs:
+    AGL: 1
+    GUT: 1
+    INT: -1
+    STR: 1
+    VIT: 1
+  Skills:
+    Finesse: 1
+    Stealth: 1
+    Knowledge: -1
+    Investigation: -1
+    Athletics: 1
+    Brute: 1
+  Powers:
+    - Attack, Weapon
+    - Attack, Vengeance
+    - Momentum
+    - Shield, Self
+    - Oath
+    - Bloodthirsty
```

Comment:
If we use this as a template and create additional PCs using this guide, would this be able to generate N character sheets in the new format reading for "type: PC"?

Reply:
Yes with the following caveat - Currently, they will all have the same inventory (unless passed as a dict to the `items` param), and the inventory will have no impact on stats.